LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadJoinAccept.cs

```diff
@@ -167,8 +163,11 @@ public override byte[] PerformEncryption(AppKey key)
             var micBytes = new byte[4];
             _ = Mic is { } someMic ? someMic.Write(micBytes) : throw new InvalidOperationException("MIC must not be null.");
 
+            Span<byte> appNonce = stackalloc byte[AppNonce.Size];
```

Comment:
[nit] why not write it to an array directly and avoid the `appNonce.ToArray()` on L170?

Reply:
Was more work but done with 6b3d9fa2837707a50297f6d54d7cb44161a665a0.